aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandlerTest.java

```diff
@@ -281,16 +349,16 @@ public void itThrowsAnyOtherCodeGuruExceptionException() {
             doThrow(CodeGuruProfilerException.builder().build()).when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
             CodeGuruProfilerException exception = assertThrows(CodeGuruProfilerException.class,
-                () -> subject.handleRequest(proxy, request, null, logger));
+                    () -> subject.handleRequest(proxy, request, null, logger));
             assertThat(exception).hasNoCause();
         }
     }
 
     private ResourceModel newResourceModel(final AgentPermissions permissions) {
         return ResourceModel.builder()
-                   .profilingGroupName(profilingGroupName)
-                   .agentPermissions(permissions)
-                   .build();
+                .profilingGroupName(profilingGroupName)
+                .agentPermissions(permissions)
+                .build();
     }
 
```

Comment:
Consider removing all of the indentation changes from this PR. It's already quite lengthy, and all these changes add a lot of noise. Perhaps set up a common code style for this project to avoid this?

Notice that there's a bit of inconsistency but almost all of our codebases (including the internal ones) use 4 spaces for indentation, so I don't recommend changing it in only a few of the files -- I'm OK with 8 as well but it should be a separate PR that changes the whole project.